elements/pfe-page-status/demo/pfe-page-status.html

```diff
@@ -0,0 +1,32 @@
+<link rel="stylesheet" href="/core/pfe-core/pfe.min.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-base.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-context.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-layouts.css">
+
+<h3>Set Status</h3>
```

Comment:
I think we should wrap this h3 and select in a <pfe-band>. That will fix the overlapping elements issue on the status demo and get these elements the correct side padding.